bedrock/exp/templates/exp/firefox/new/download.html

```diff
@@ -2,50 +2,518 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% from "macros.html" import google_play_button, fxa_email_form with context %}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% set show_firefox_join_modal = ftl_has_messages('firefox-desktop-download-youve-already-got-the-browser',
+                                                  'firefox-desktop-download-watch-for-hackers-with',
+                                                  'firefox-desktop-download-just-download-the-browser',
+                                                  'firefox-desktop-download-get-more-from-firefox') %}
+
 {% extends "exp/firefox/new/base.html" %}
 
-{% block page_title_prefix %}Download Firefox — {% endblock %}
+{% block extrahead %}
+  {{ super() }}
 
-{% block page_title %}Free Web Browser{% endblock %}
+  <!--[if IE 9]>
+    {{ css_bundle('firefox_desktop_download_ie9') }}
```

Comment:
Are you OK with using the source originals for these assets, vs making /exp/ ones?

Reply:
yes, I think that is fine for IE styles.